README.md

```diff
@@ -5,6 +5,7 @@ to write software that makes use of AmazonQLDB.
 
 [![Latest Version](https://img.shields.io/pypi/v/pyqldb.svg)](https://pypi.python.org/pypi/pyqldb)
 [![Documentation Status](https://readthedocs.org/projects/amazon-qldb-driver-python/badge/?version=latest)](https://amazon-qldb-driver-python.readthedocs.io/en/latest/?badge=latest)
+![Python application](https://github.com/awslabs/amazon-qldb-driver-python/workflows/Python%20application/badge.svg)
```

Comment:
Does this magically update? The phrase should just be "build", not "Python application"

Reply:
I generated this badge in the workflows [page](https://github.com/awslabs/amazon-qldb-driver-python/actions?query=workflow%3A%22Python+application%22).

It takes the name of the workflow and I am not able to edit the name of the workflow.

Here is an [Ion dotnet project](https://github.com/amzn/ion-dotnet) with the badge for reference.

Reply:
That Ion one looks different... it has a second set of parentheses... you sure yours is right?

Reply:
I did not hyperlink the status badge, I've updated it and verified [here](https://github.com/awslabs/amazon-qldb-driver-python/blob/feature/shell-changes/README.md)